python/test/unit/mesh/test_higher_order_mesh.py

```diff
@@ -22,6 +19,11 @@
 from ufl import dx
 
 
+scipy = pytest.importorskip("scipy")
```

Comment:
I think we could make SciPy a test dependency.

Reply:
I'm not sure how to make SciPy a test dependency.
Using requirements files, maybe? eg: `test_requirements.txt`